lib/models/addon.js

```diff
@@ -412,16 +412,16 @@ var Addon = CoreObject.extend({
 
     @public
     @method included
-    @param {EmberApp} app The application object
+    @param {EmberApp|Addon} parent The parent object that included this addon
 
     @example
     ```js
-    included: function() {
+    included: function(parent) {
       this.import(somePath);
     }
     ```
   */
-  included: function(/* app */) {
+  included: function(/* parent */) {
```

Comment:
The generic `parent` value here is probably for the best as we're looking at have `Engines`, `Addons`, and `Applications` soon.